packages/admin-rte/src/core/Rte.tsx

```diff
@@ -86,13 +86,21 @@ export interface CometAdminRteThemeProps {
         buttonBorderHover?: React.CSSProperties["color"];
         buttonBorderDisabled?: React.CSSProperties["color"];
     };
+    editor?: {
+        minHeight?: number;
+    };
 }
 
-export interface IProps extends CometAdminRteThemeProps {
+export interface IProps {
     value: EditorState;
     onChange: OnEditorStateChangeFn;
     options?: IOptions;
     disabled?: boolean;
+    theme?: CometAdminRteThemeProps;
+    /**
+     * @deprecated Use theme.colors instead.
+     */
+    colors?: any;
```

Comment:
Passing colors didn't work somehow (at least I couldn't get it to work), so I do not handle the deprecated-case as I think it's broken anyway.

Reply:
I think we can/should remove this depreacted prop for comet-admin 2